src/renderer/helpers/api/local.js

```diff
@@ -68,7 +69,14 @@ export async function getLocalSearchSuggestions(query) {
     searchSuggestionsSession = await createInnertube()
   }
 
-  return await searchSuggestionsSession.getSearchSuggestions(query)
+  const searchCharLimit = store.getters.getSearchCharacterLimit
+
+  if (query.length <= searchCharLimit) {
+    return await searchSuggestionsSession.getSearchSuggestions(query)
+  } else {
+    // There's an event handler on the search input so avoid displaying an exception
+    console.error('Query is over ' + searchCharLimit + ' characters')
```

Comment:
Similar to another file, these lines should be added to the top of function with early return to avoid calling `createInnertube` unnecessarily

Reply:
I've made the changes requested

Reply:
@absidue I guess we should throw an error here instead of returning? I am not quite sure

Reply:
The problem is this code is called when typing inside the search bar by the event handler. Due it's usage you might see it quite a few times. The assumption I'm making is they are going to hit the search button eventually which will then show the proper error message popup (Inside Search.js).